packages/api/app/index.js

```diff
@@ -11,7 +11,9 @@ const morgan = require('morgan');
 const awsServerlessExpress = require('aws-serverless-express');
 const awsServerlessExpressMiddleware = require('aws-serverless-express/middleware');
 
-const { services } = require('@cumulus/aws-client');
+const { getJsonS3Object } = require('@cumulus/aws-client/S3');
+const Logger = require('@cumulus/logger');
+const log = new Logger('@cumulus/api/handler');
```

Comment:
```suggestion
const log = new Logger('@cumulus/api/index');
```
seems the context is for the whole file.